src/main/java/com/notably/logic/commands/HelpCommand.java

```diff
@@ -11,9 +14,13 @@
     public static final String COMMAND_WORD = "help";
     public static final String COMMAND_SHORTHAND = "h";
 
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `HelpCommand`!